numpy_financial/_financial.py

```diff
@@ -825,6 +827,40 @@ def irr(values, *, guess=None, tol=1e-12, maxiter=100, raise_exceptions=False):
     return np.nan
 
 
+@nb.njit(parallel=True)
+def _npv(rates, cashflows, result):
+    r"""Native version of the ``npv`` function.
+
+    Warnings
+    --------
+    For internal use only, note that this function performs no error checking.
```

Comment:
Documentation still needs to be improved; this is the workhorse powering the `npv` function.